spec/requests/trip_destroy_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Trip destruction', type: :request do
+  it 'has a valid factory' do
+    expect(FactoryBot.create(:trip)).to be_valid
+  end
+
+  let!(:trip) do
+    FactoryBot.create(:trip)
+  end
+
+  let(:trip_destroy) { delete "/trips/#{trip.id}" }
+
+  it 'returns 204' do
+    trip_destroy
+    expect(response).to have_http_status(204)
+  end
+
+  it 'destroys a trip' do
```

Comment:
You could check count change and returned status in one spec.

Reply:
merged